locales/en-US/strings.properties

```diff
@@ -136,6 +136,7 @@ edit_topsites_unpin_button=Unpin this site
 edit_topsites_edit_button=Edit this site
```

Comment:
Hmm... let's try adding a plain `# remove after 60` comment before each string that's now unused